src/servidor/CadastroServidorPage.tsx

```
@@ -68,37 +71,75 @@ export function CadastroServidorPage() {
             cpf: "",
             siape: "",
-            emailInstitucional: validatedEmail || '',
+            email: "",
             telefone: "",
             departamento: "",
             id: undefined,
         },
     })
+    const { reset } = form
+
+    useEffect(() => {
```

ESLint

[error] Delete `··`

(prettier/prettier)